tests/services/label.service.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { LabelService } from '../../src/app/core/services/label.service';
+
+let labelService: LabelService;
```

Comment:
Extract all constants in a separate file

Reply:
I have made the necessary changes